tlatools/org.lamport.tlatools/src/pcal/PcalTLAGen.java

```diff
@@ -1843,14 +1843,14 @@ private void GenVarsAndDefs(Vector globals, Vector procs, Vector processes, TLAE
                 vars.addElement(decl.var);
             }
         if (! ParseAlgorithm.omitPC) {
```

Comment:
Have you considered keeping `addElement` and moving the if block up instead? Additionally, it would be helpful if you could add a comment explaining the rationale behind why we want `pc` to occur first.